week-1/Homework/mandatory/1-syntax-errors.js

```diff
@@ -1,19 +1,25 @@
 // There are syntax errors in this code - can you fix it to pass the tests?
 
-function addNumbers(a b c) {
+function addNumbers(a, b, c) {
     return a + b + c;
 }
 
-function introduceMe(name, age)
-return "Hello, my name is " + name "and I am " age + "years old";
+function introduceMe(name, age){
+return "Hello, my name is " + name + " and I am " + age + " years old";
+
+}
+
+
 
 function getRemainder(a, b) {
-    remainder = a %% b;
+    remainder = a % b;
```

Comment:
Works great but you forgot something in front of this `remainder` variable and now it's implicitly a global variable 😱 Do you know what it is?